### ISSUE TITLE
[Keyboard navigation - ADF - Azure data factory] Tooltip is not visible through keyboard for Azure data factory control.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3839]
**User Experience:**
User will find difficulty to identify the purpose of visual icon if tooltip is not visible for that visual icon. Not only this, they won't be able to get the relevant details of such control.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it  

**Repro Steps:​**
1. Launch Azure Storage Explorer app.
2. Navigate to visual icon Add data factory control at left panel.
3. Verify tooltip of visual icon Add data factory.

**Actual Result:**
Tooltip is not visible through keyboard for visual icon Add data factory control and due to this end user is unable to get the relevant details of such icon.

**Expected Result:**
Tooltip should be visible through keyboard for visual icon Add data factory control so that end user could easily get the relevant details of such icon.

**Note:**
Tooltip is also not visible for Minimize/maximize/Restore down and close controls through keyboard.

[MAS2.1.1_Azure data factory.zip](https://github.com/microsoft/AzureStorageExplorer/files/5636626/MAS2.1.1_Azure.data.factory.zip)
